package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@koopjs/provider-file-geojson",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "Simple provider for serving file-based GeoJSON",
-  "main": "dist/index.js",
+  "main": "src/index.js",
```

Comment:
This needs to be left as `dist/index.js` because we run `npm run compile` before the publish to npm which packages everything into the `dist` directory.

Reply:
Happy to roll this back, however, the code is not saved to GitHub, so I can't use the package directly until it gets published to NPM.